invenio_deposit/views/ui.py

```diff
@@ -68,27 +69,25 @@ def index():
         return render_template(current_app.config['DEPOSIT_UI_INDEX_TEMPLATE'])
 
     @blueprint.route('/deposit/new')
-    @blueprint.route('/deposit/<string:depid>/new')
+    @blueprint.route('/deposit/new/<string:deposit_type>')
```

Comment:
Please lets add a 'DEPOSIT_REGISTER_UI_ENDPOINT' (or smthg) config flag, for the creation or not of this 'New' endpoint @egabancho @jirikuncar . We probably won't use it